src/puppetlabs/puppetdb/pdb_routing.clj

```diff
@@ -44,29 +44,29 @@
          (partition
           2
           ;; The remaining get-shared-globals args are for wrap-with-globals.
-          ["/meta" (meta/build-app db-cfg defaulted-config)
+          ["/" (fn [req]
```

Comment:
Nope it used to be handled in a compojure app here https://github.com/puppetlabs/puppetdb/pull/1940/files#diff-e58d11151e2bd18dfb7e14b63a5e2c70L61